.github/workflows/create-pr.yml

```diff
@@ -4,8 +4,22 @@ on:
     branches:
       - feat/bot-*
 jobs:
+  get-pull-request:
+    runs-on: ubuntu-latest
+    outputs:
+      pr_number: ${{ steps.pr.outputs.pr_number }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get Pull Request Number
+        id: pr
+        run: echo "::set-output name=pull_request_number::$(gh pr view --json number -q .number || echo "")"
```

Comment:
this is the deprecated way of setting outputs, 

Very pedantic but your cloning the workflow twice, you could only run `build` if the output of `create-pull-request` was a new pull request? similar to how release please?